lib/ansible/modules/hostname.py

```diff
@@ -1001,6 +1001,12 @@ class PopHostname(Hostname):
     strategy_class = DebianStrategy
 
 
+class RockyLinuxHostname(Hostname):
+    platform = 'Linux'
+    distribution = 'RockyLinux'
```

Comment:
Please change to 'Rocky' -- The ID is simply `Rocky`
```suggestion
    distribution = 'Rocky'
```